package.json

```diff
@@ -21,7 +21,7 @@
     "compile": "env NODE_ENV=production concurrently yarn:compile:*",
     "compile:main": "yarn run webpack --config webpack/main.ts",
     "compile:renderer": "yarn run webpack --config webpack/renderer.ts",
-    "compile:extension-types": "yarn run webpack --config webpack/extensions.ts",
+    "compile:extension-types": "yarn run tsc --project tsconfig.extension-api.json",
```

Comment:
I don't see the motivation of this commit. Better commit message in the future? :)

Reply:
Will try and improve my commit messages for the future.

IIRC this change was two fold:

1. I was having problems with the previous command and some plugins that were being used for some reason
2. The output of this is cleaner and no longer produces any JS at all.

Reply:
Ok. Resolved.